temba/flows/tests.py

```diff
@@ -4180,6 +4180,20 @@ def test_webhook(self):
             self.assertIsNone(value)
             self.assertEquals("1001", incoming.text)
 
+        with patch('requests.post') as mock:
+            mock.return_value = MockResponse(400, '{ "error": "400", "message": "Missing field in request" }')
+            rule_step.run.fields = None
+            rule_step.run.save()
+
+            (match, value) = webhook.find_matching_rule(webhook_step, run, incoming)
+            (match, value) = rules.find_matching_rule(rule_step, run, incoming)
+            self.assertIsNone(valid_uuid, match.uuid)
```

Comment:
@chris-erickson 
First `assertIsNone` takes one argument, I think here you need `assertEquals` 

And you need to make sure you have a rule that will be matched on the ruleset https://github.com/rapidpro/rapidpro/pull/632/files#diff-0f24266e2d8c853528ec9af33da0614bR4030

so valid_uuuid is the rule when the response contains Valid

the other rule matches when we have 'Invalid'

1. You need a predictable uuid for the invalid rule 
2. You need invalid in the response or better a new rule in the ruleset that